packages/@o3r/dev-tools/schematics/ng-add/index.ts

```diff
@@ -1,12 +1,11 @@
-import { noop } from '@angular-devkit/schematics';
-import type { Rule } from '@angular-devkit/schematics';
+import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
 
 /**
  * Add Otter dev-tools to an Angular Project
- *
- * @param options
  */
 export function ngAdd(): Rule {
-  /* ng add rules */
-  return noop();
+  return (tree: Tree, context: SchematicContext) => {
+    context.logger.warn('This package is deprecated and will no longer be updated as of v12.');
```

Comment:
I would add the package name on the message to be sure that the warning is relevant even if it is part of multiple commands
```suggestion
    context.logger.warn('@o3r/dev-tools package is deprecated and will no longer be updated as of v12.');
```